### PR TITLE
Add logic for deploy functions

### PR DESCRIPTION
**Pull request by Flanker32**
Add logic for deploy functions 

#### Known issues
Can't redirect info messages to IntelliJ Console

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/functions/RunFunctionAction.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import org.apache.commons.lang.StringUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.microsoft.intellij.runner.functions.AzureFunctionsConstants.FUNCTION_JAVA_LIBRARY_ARTIFACT_ID;
+
+public class RunFunctionAction extends AzureAnAction {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/functions/RunFunctionAction.java]: too many duplicate codes from deploy

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/localrun/FunctionRunState.java [hunk 2]
```diff
@@ -153,7 +153,7 @@ private ProcessBuilder getRunFunctionCliProcessBuilder(File stagingFolder, int f
         final String debugConfiguration = String.format("\"-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=%s\"", debugPort);
         String[] command = {funcPath, "host", "start", "--port", String.valueOf(funcPort)};
         if (isDebugMode()) {
-            command = ArrayUtils.addAll(command, new String[] { "--language-worker", "--", debugConfiguration });
+            command = ArrayUtils.addAll(command, new String[]{"--language-worker", "--", debugConfiguration});
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/localrun/FunctionRunState.java]: new String[] {

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/functions/RunFunctionAction.java (hunk 1 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/functions/RunFunctionAction.java]: Extract logic of getting runtime configuration to util method

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/localrun/FunctionRunState.java (hunk 2 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/localrun/FunctionRunState.java]: Fixed

**Pull request merged**